Documentations/dev_guide.md

```diff
@@ -0,0 +1,119 @@
+# DataHub Docker Stack: Development Guide
+
+## Docker Concepts
+
+Before listing the commands we use, we'd like to give a brief introduction of Docker concepts.
+
+- **Docker**: Docker is a set of platform as a service (PaaS) products that use OS-level virtualization to deliver software in packages called containers. In short, it's a platform that provide you with containerized/isouldated software/system.
+- **Docker Image**: Image is a static, immutable template that defines the behavior of your software/system. Most of the time, we 'build' an image.
+- **Docker Container**: Container is a runtime instance. Most of the time, we 'run' or 'start' a container.
+- **Dockerhub**: Dockerhub is a place to store your Docker images just like Github stores your code.
+
+## Docker Commands
+
+Here are the commands (signature + example) you will use often:
+
+- image-related
+
+```bash
+# Build an image from a local Dockerfile
+# <path> is where the Dockerfile stays in
+$ docker built -t <image_name>:<tag> <path>
```

Comment:
It should be docker build not built right

Reply:
Yeah definitely!